firefox-ios/Client/Glean/glean_index.yaml

```diff
@@ -6,6 +6,7 @@ tag_files:
   - firefox-ios/Client/Glean/tags.yaml
 # The following section must be last in the file and is autogenerated by FirefoxTelemetryUtility.sh
 metrics_files:
+  - firefox-ios/Client/Glean/probes/ai.yaml
```

Comment:
Is this the most descriptive name for this telemetry? If I just look at this, I kinda have no idea what it really is. Perhaps a better name would be `summarizer` or `ai_summary` or something. `ai` is a big category, and we'll likely have more AI stuff in the app in the future, and then we don't want to mix these things up.

Reply:
Updated in https://github.com/mozilla-mobile/firefox-ios/pull/28453/commits/d14bd393ed9f6ca3a74e2c2ce055a58258f399f4 plus added a tag for summarizer